Client/src/composables/useClickOutside.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @param {*} element The Root element for which clicking outside will trigger callbackFunction
+ * @param {*} callbackFunction The function to call when the user clicks outside of the element
+ * @returns
+ */
+
+import { onMounted, onBeforeMount } from 'vue';
+
+export function useClickOutside(element, callbackFunction) {
+  if (!element) return;
+
+  let listener = (event) => {
+    if (event.target !== element.value && event.composedPath().includes(element.value)) {
+      return;
+    }
+
+    if (typeof callbackFunction === 'function') {
+      callbackFunction();
+    }
+  };
+
+  onMounted(() => {
+    window.addEventListener('click', listener);
```

Comment:
Ik ben benieuwd of je maar 1 event voor de window krijgt of meerdere als je meerdere components gebruik laat maken van `useClickOutside`. Het mooiste zou zijn maar 1 event waar meerdere components gebruik van kunnen maken.

Reply:
Meerdere componenten luisteren naar hetzelfde event.